store/tikv/backoff.go

```diff
@@ -102,8 +102,13 @@ func NewBackoffFn(base, cap, jitter int) func(ctx context.Context) int {
 		logutil.Logger(context.Background()).Debug("backoff",
 			zap.Int("base", base),
 			zap.Int("sleep", sleep))
+
+		realSleep := sleep
+		if maxSleepMs >= 0 && realSleep > maxSleepMs {
```

Comment:
When the maxSleepMs will be 0?

Reply:
maxSleepMs never should be 0 in current usage, but backoff is a common function and we add new maxSleepMs Param, so we should take care about it and in logic "force sleep zero" is useful.